### PR TITLE
Fix typos found by codespell

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
# Description

Fix typos found by [codespell](https://github.com/codespell-project/codespell).

# Testing

\-

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [x] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

**Comment by JacobBarthelmeh** [comment]
ok to test

**Comment by DimitriPapadopoulos** [comment]
I don't have access to the Jenkins tests. Please advise.

**Comment by JacobBarthelmeh** [comment]
Looking into it @DimitriPapadopoulos , we have some external interop tests too that could have had an issue. Retest this please Jenkins.

File: src/internal.c [hunk 1]
```diff
@@ -6309,7 +6309,7 @@ static void InitSuites_EitherSide(Suites* suites, ProtocolVersion pv, int keySz,
         word16 haveFalconSig, word16 haveDilithiumSig, word16 haveAnon,
         int side)
 {
-    /* make sure server has DH parms, and add PSK if there */
+    /* make sure server has DH paras, and add PSK if there */
```
Comment (JacobBarthelmeh) [inline src/internal.c:6312]: believe that this should be params (short for parameters)

Comment (DimitriPapadopoulos) [reply src/internal.c:6312]: Of course, that's a typo "fixed" by another typo :smile:

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**